software/contrib/cvecorder.py

```diff
@@ -271,7 +272,7 @@ def saveState(self):
                     print(f'[{attempts}] Error: Memory allocation failed, retrying: {e}')
                 if self.debugLogging:
                     self.writeToDebugLog(f"[saveState] Error: Memory allocation failed, retrying: {e}")
-                    #print(micropython.mem_info("level"))
+                    print(micropython.mem_info("level"))
```

Comment:
Same here

Reply:
Same thing here, this is behind the `debugLogging` which is manually enabled.